Reference/Searching/Examine/Quick-Start/index.md

```diff
@@ -159,71 +174,75 @@ The final template looks like this:
                                     {
-                                        var node = Umbraco.Content(result.Id);
-                                        <li>
-                                            <a href="@node.Url">@node.Name</a>
-                                        </li>
+                                        if (result.Id != null)
+                                        {
+                                            var node = Umbraco.Content(result.Id);
+                                            <li>
+                                                <a href="@node.Url()">@node.Name</a>
+                                            </li>
+                                        }
                                     }
-                                }
-                            </ul>
-                        }
-                        else
-                        {
-                            <p>No results found for query @searchTerm</p>
+                                </ul>
+                            }
+                            else
+                            {
+                                <p>No results found for query @searchTerm</p>
+                            }
                         }
+                        return;
                     }
-                    return;
                 }
-            }
+            </div>
         </div>
         <div class="employee-grid">
-            @foreach (ContentModels.Person person in Model.Children)
+            @foreach (Person person in Model.Children<Person>())
             {
+
                 <div class="employee-grid__item">
-                    <div class="employee-grid__item__image" style="background-image: url('@person.Photo.Url')"></div>
+                    <div class="employee-grid__item__image" style="background-image: url('@person.Photo?.Url()')"></div>
                     <div class="employee-grid__item__details">
                         <h3 class="employee-grid__item__name">@person.Name</h3>
                         @if (!string.IsNullOrEmpty(person.Email))
                         {
                             <a href="mailto:@person.Email" class="employee-grid__item__email">@person.Email</a>
                         }
                         <div class="employee-grid__item__contact">
-                            @SocialLink(person.FacebookUsername, "Facebook")
-                            @SocialLink(person.TwitterUsername, "Twitter")
-                            @SocialLink(person.LinkedInUsername, "LinkedIn")
-                            @SocialLink(person.InstagramUsername, "Instagram")
+                            @{ SocialLink(person.FacebookUsername, "Facebook"); }
+                            @{ SocialLink(person.TwitterUsername, "Twitter"); }
+                            @{ SocialLink(person.LinkedInUsername, "LinkedIn"); }
+                            @{ SocialLink(person.InstagramUsername, "Instagram"); }
```

Comment:
Are you sure about this?

I don't see where the SocialLink() is coming from.

Reply:
Sorry I missed your review here Damiaan!

I haven't touched this for any Examine related reasons. 

The SocialLink() method is part of [the starter kit](https://github.com/umbraco/The-Starter-Kit/blob/v3.0.0/src/Umbraco.SampleSite.Website/Views/People.cshtml#L42-L45) which this code is extending, don't remember why I had to add this razor code around them, but if it works without we can just remove it - I seem to remember running into some issues with it throwing errors though.